infrastructure/provision-server/hetzner-cloud-empty-server/multi-node/main.tf

```diff
@@ -25,12 +26,12 @@ resource "hcloud_server" "worker" {
     ipv4_enabled = true
     ipv6_enabled = true
   }
-  network {
-    network_id = var.private_network_id
-    # IP Used by the master node, needs to be static
-    # Here the worker nodes will use 10.0.1.1 to communicate with the master node
-    ip         = var.ip
-  }
+  # network {
+  #   network_id = var.private_network_id
+  #   # IP Used by the master node, needs to be static
+  #   # Here the worker nodes will use 10.0.1.1 to communicate with the master node
+  #   ip         = var.ip
+  # }
```

Comment:
maybe smart to add comment here and below as well, to explain why this is commented out